docs/overview.rst

```diff
@@ -41,8 +42,9 @@ frequency of ~66 Hz. If you want to use the 200 Hz gaze data from pupil cloud, y
 recording with your pupil invisible companion device simultaneously with the LSL recording, and use the ``lsl.time_sync.*``
 events generated by the relay to align you data streams post-hoc.
 
-**Important:** If you want to do the post-hoc alignment of LSL data and cloud data, you must also subscribe to the LSL
-event stream and make sure that at least two events are contained in your recording.
+.. Important::
+  If you want to do the post-hoc alignment of LSL data and cloud data, you must also subscribe to the LSL
+  event stream and make sure that at least two events are contained in your recording.
```

Comment:
In rst, it is best practice to left-align blocks with their starting marker. Please apply this suggestion to the other blocks as well.
```suggestion
.. Important::
   If you want to do the post-hoc alignment of LSL data and cloud data, you must also subscribe to the LSL
   event stream and make sure that at least two events are contained in your recording.
```
For details, see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html
> Several constructs begin with a marker, and the body of the construct must be indented relative to the marker. For constructs using simple markers ([bullet lists](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists), [enumerated lists](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#enumerated-lists)), the level of indentation of the body is determined by the position of the first line of text. For example:

```
- This is the first line of a bullet list
  item's paragraph.  All lines must align
  relative to the first line.

      This indented paragraph is interpreted
      as a block quote.

  Another paragraph belonging to the first list item.

 Because it is not sufficiently indented,
 this paragraph does not belong to the list
 item (it's a block quote following the list).
 ```